<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Диспетчер Т-70   v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Служебные</string>
       </attribute>
       <widget class="QSlider" name="horizontalSlider_4">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>800</y>
          <width>411</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="sliderPosition">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_5">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>850</y>
          <width>411</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="text_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>T8:</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>281</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>T2</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>T3</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>T5</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>T7</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>281</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>T1</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>T6</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>80</y>
          <width>251</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Питание включено (Включение КУ)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="text_10">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>T10</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>T9</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>T4</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>303</x>
          <y>316</y>
          <width>460</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>Входные данные</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_11">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>260</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>T11</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Электро</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="16" column="0">
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0">
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QLabel" name="label_42">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="20" column="0">
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="20" column="1">
         <widget class="QLabel" name="label_43">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="21" column="0">
         <widget class="QLabel" name="label_44">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="21" column="1">
         <widget class="QLabel" name="label_45">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Тормозная</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Шасси</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Гидро</string>
       </attribute>
       <widget class="QLabel" name="text_3004">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Текст_4</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3002">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Текст_2</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3001">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Текст_1</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3005">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Текст_5</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3003">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Текст_3</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3007">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>180</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Текст_7</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3008">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Текст_8</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3009">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>220</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Текст_9</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3010">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>240</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Текст_10</string>
        </property>
       </widget>
       <widget class="QLabel" name="text_3006">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>160</y>
          <width>271</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Текст_6</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Пневмо</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>630</y>
          <width>91</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Катапульт.</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Мех.Крыла</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>КСКВ</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Освещение</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>САС вход</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>САС вых.</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>ПОС</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_14">
       <attribute name="title">
        <string>СРД</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
